CHANGELOG.adoc

```diff
@@ -28,6 +28,7 @@
 - https://github.com/eclipse-sirius/sirius-web/issues/4195[#4195] [sirius-web] Prevent objects disappearing when dropped on themselves (or a descendant) in the _Explorer_.
 
 === New Features
+- https://github.com/eclipse-sirius/sirius-web/issues/4190[#4190] [core] Display the icon of the representation in the workbench tab
```

Comment:
I think we should add a Breaking Changes to indicate that the graphql API has changed and that `RepresentationMetadata` must now expose a list of icons

Reply:
Done